grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -341,30 +342,32 @@ class StockMovementApiController {
     }
 
     def exportPickListItems() {
```

Comment:
As I mentioned on the call, these methods don't really belong in the API, so let's move them to StockMovementController, or probably more appropriately, to PicklistController.